.gitignore

```diff
@@ -138,3 +138,7 @@ dmypy.json
 .DS_Store
 .vscode/
 .idea/
+.env
+CACHE/
+db.sqlite3
+.gitignore
```

Comment:
all these changes are not related and not useful

it would be great if you can remove them